### PR TITLE
ignore JDK dependencies

### PR DESCRIPTION
**Pull request by Obinnabii**
To get Java language support in VSCode, I need to use ```JDK```. I don't want these dependencies to be seen accross systems using this repo because I don't know how it would affect the gcloud system.

### CODE REVIEW

**Comment by billyeh** [comment]
Maybe just copy the gitignore for VSCode JDK? https://github.com/redhat-developer/vscode-java/blob/master/.gitignore

Hope the VSCode setup works! :)

**Comment by Obinnabii** [comment]
oh yeah definitely! I didn't even know that was a thing lol. I'll look into that. thanks!

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+.DS_Store
+.vscode-test
+undefined
+target
+dist
+bin/
+.settings
+.classpath
+.project
+© 2020 GitHub, Inc.
```
Comment (billyeh) [inline .gitignore]:
```suggestion
```
Not sure if we need this in here :)

Comment (Obinnabii) [reply .gitignore]: hahah my bad

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,15 @@
 target/
 .idea
+out
+server
+node_modules
+*.vsix
+.DS_Store
+.vscode-test
+undefined
+target
```
Comment (billyeh) [inline .gitignore]:
```suggestion
```
already in line 1 :)

**Review by billyeh — APPROVED** [review]

**Comment by billyeh** [comment]
Nice! Yeah generally for new dependencies/code/projects they'll have gitignore examples. Impressed that you came up with the list yourself though.

**Comment by Obinnabii** [comment]
Thanks!

**Pull request merged**